docs/pages/web/dropdown/show_case.js

```diff
@@ -0,0 +1,536 @@
+// @flow strict
+import { type Node } from 'react';
+import Page from '../../../docs-components/Page.js';
+import PageHeader from '../../../docs-components/PageHeader.js';
+import MainSection from '../../../docs-components/MainSection.js';
+
+export default function DropdownShowCase(): Node {
+  return (
+    <Page title="Dropdown - Show case">
+      <PageHeader badge="pilot" name="Dropdown" type="guidelines" />
+      <MainSection name="Examples">
+        <MainSection.Subsection>
+          <MainSection.Card
+            cardSize="lg"
+            title="More items then height of viewport"
```

Comment:
https://www.loom.com/share/3f1b8347489e4e1385a736ff3d5477b1 First example

Reply:
https://user-images.githubusercontent.com/16820917/202543564-68ff978e-5e3d-4763-b343-d5c55d9f6904.mp4

Reply:
This is still rendering partially offscreen…is that expected?

![Screen Shot 2022-12-07 at 6 07 01 PM](https://user-images.githubusercontent.com/12059539/206338945-8d11d3df-7b6b-4464-a2c5-0f6b03a8cd22.png)

Reply:
Is not out, because `this screen is scrollable`, so you can access the items, the current problem is when you go out and not access that area. 
And this problem was solved.